src/cheri_insts.sail

```diff
@@ -1639,13 +1758,74 @@ function clause execute AUIPCC(imm, cd) = {
 }
 
 union clause ast = LoadCapImm : (regidx, regidx, bits(12))
+/*!
+ * In integer mode, capability register *cd* is replaced with the capability
+ * located in memory at **DDC**.**address** $+$ *rs1* $+$ *imm*, and if
+ * **DDC**.**perms** does not grant **Permit_Load_Capability** then
+ * *cd*.**tag** is cleared. In capability mode, capability register *cd* is
+ * replaced with the capability located in memory at *cs1*.**address** $+$
+ * *imm*, and if *cs1*.**perms** does not grant **Permit_Load_Capability** then
+ * *cd*.**tag** is cleared.
+ *
+ * ## Exceptions
+ *
+ * In integer mode, an exception is raised if:
+ *   - **DDC**.**tag** is not set.
+ *   - **DDC** is sealed.
+ *   - **DDC**.**perms** does not grant **Permit_Load**.
+ *   - **DDC**.**address** $+$ *rs1* $+$ *imm* $\lt$ **DDC**.**base**.
+ *   - **DDC**.**address** $+$ *rs1* $+$ *imm* $+$ **CLEN** $\gt$
```

Comment:
Is it obvious that CLEN is in bytes here where normally it's bits? Are we okay for context to make this clear or should we have a disambiguation (e.g. XLEN vs xlen)?

Reply:
Oh that's a good point...

Reply:
I could do `CLEN / 8`?

Reply:
Hmm, I suppose. CLEN / BITS_PER_BYTE seems overly verbose...

Reply:
Not particularly keen to encourage things akin to C's `CHAR_BITS`... :joy: